### ISSUE TITLE
Autodoc errors for documentation of cea/optimization

### ISSUE THREAD

**Issue by Jack-Hawthorne** [architecture-building-systems/CityEnergyAnalyst#1727]
The following errors are raised when creating documentation for cea/optimization:
```
C:\Users\Jack\Documents\GitHub\CityEnergyAnalyst\docs\modules\cea.optimization.rst:12: WARNING: toctree contains reference to nonexisting document u'modules/cea.optimization.individual_evaluation'

c:\users\jack\documents\github\cityenergyanalyst\cea\optimization\optimization_main.py:docstring of cea.optimization.optimization_main.moo_optimization:3: WARNING: Unexpected indentation.

c:\users\jack\documents\github\cityenergyanalyst\cea\optimization\master\generation.py:docstring of cea.optimization.master.generation.random_sample:10: WARNING: Unexpected section title.

Parameters
----------
c:\users\jack\documents\github\cityenergyanalyst\cea\optimization\master\generation.py:docstring of cea.optimization.master.generation.random_sample:17: WARNING: Unexpected section title.

Returns
-------
c:\users\jack\documents\github\cityenergyanalyst\cea\optimization\master\generation.py:docstring of cea.optimization.master.generation.random_sample:23: WARNING: Unexpected section title.

Examples
--------
```
Looking into the issues now.
